opm/simulators/DeferredLogger.cpp

```diff
@@ -86,6 +86,12 @@ namespace Opm
         for (const auto& m : messages_) {
             OpmLog::addTaggedMessage(m.flag, m.tag, m.text);
         }
+        messages_.clear();
```

Comment:
This could **maybe** be a call to the
```C++
clearMessages()
```
member function instead&mdash;just in case `clearMessages()` ever evolves to do something more/other than `messages_.clear()`.

Reply:
I considered it, but decided against (why I am do not remember...)

Reply:
>> This could **maybe** be a call to the`clearMessages()` member function instead
>
> I considered it, but decided against [it]

I see. Well, in that case I have no further comments and will merge as is.